### ISSUE TITLE
accessibility lint + security risk

### ISSUE THREAD

**Issue by tattwamasii2p** [nonlin-lin-chaos-order-etc-etal/myzone_now_sh_GPLv3#6]
`warn ESLintError: `
`/home/user/git/zone/src/pages/index.js`
`  18:9   warning  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby                   jsx-a11y/accessible-emoji`
`  20:7   warning  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby                   jsx-a11y/accessible-emoji`
`  21:7   warning  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby                   jsx-a11y/accessible-emoji`
`  26:60  warning  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby                   jsx-a11y/accessible-emoji`
`  46:25  warning  Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener  react/jsx-no-target-blank`
